src/fcmcts/fcmcts.jl

```diff
@@ -0,0 +1,307 @@
+
+
+@with_kw mutable struct FCMCTSSolver <: AbstractMCTSSolver
```

Comment:
Note to self: Add docu ala https://github.com/JuliaPOMDP/MCTS.jl/blob/a24e92d108853a0410ac0a61f64d4e53a8623e71/src/vanilla.jl#L1-L52